Entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export PATH=$PATH:/opt/keycloak/bin/
+
+if [ "$1" = "import" ]; then
+    echo "Performing import..."
+    mkdir -p /home/default/TEMP_IMPORT_DIR
+    cp -r /home/default/IMPORT_DIR/* /home/default/TEMP_IMPORT_DIR
+    export WORK_DIRECTORY=/home/default/TEMP_IMPORT_DIR
+    user-import.sh
+elif [ "$1" = "export" ]; then
+    echo "Performing export..."
+    export WORK_DIRECTORY=/home/default/EXPORT_DIR
+    user-export.sh
```

Comment:
Move user-export.sh and  user-import.sh and accordingly change path 

also i would add an "help" option

Reply:
i didnt understand
`
Move user-export.sh and user-import.sh and accordingly change path

`

Reply:
Goes with the previous comment 
user-export. sh and user-import. sh should not be in the root folder as you don't expect users to use them directly

Reply:
you mean in the repo?
because the code you commented about is relevant only inside the container,